### PR TITLE
GUACAMOLE-25: Add audio input support to RDP

### PR DESCRIPTION
**Pull request by mike-jumper**
Yes, it is a very large change.

### CODE REVIEW

File: src/protocols/rdp/rdp_disp.h [hunk 1]
```diff
@@ -102,7 +103,7 @@ void guac_rdp_disp_free(guac_rdp_disp* disp);
  *
  * @param context The rdpContext associated with the active RDP session.
  */
-void guac_rdp_disp_load_plugin(rdpContext* context);
+void guac_rdp_disp_load_plugin(rdpContext* context, guac_rdp_dvc_list* list);
```
Comment (jmuehlner) [inline src/protocols/rdp/rdp_disp.h]: Document new param?

**Comment by jmuehlner** [comment]
Looks ok to me - @flangelo any thoughts?

File: src/protocols/rdp/rdp_disp.h (hunk 1 above)
Comment (mike-jumper) [inline src/protocols/rdp/rdp_disp.h]: Oops. Will add.

Comment (mike-jumper) [inline src/protocols/rdp/rdp_disp.h]: Looks like I missed this for `guac_rdp_audio_load_plugin()`, too.

**Comment by flangelo** [comment]
Looks good to me. Great work @mike-jumper !

**Comment by mike-jumper** [comment]
Thanks @flangelo !

**Pull request merged**